features/guidance/anticipate-lanes.feature

```diff
@@ -623,40 +623,7 @@ Feature: Turn Lane Guidance
             """
                  /a\
             x – b   d – y
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
+                 \ /
```

Comment:
can you give a bit more context for the test-changes? The reduction in distance is quite severe in a those two. Do we do much less lane anticipation now?